apps/settings/js/modules/bluetooth/bluetooth_context.js

```diff
@@ -749,6 +771,35 @@ define(function(require) {
     },
 
     /**
+     * Given paired, address properties to find out device element 
+     * from remote/paired devices list.
+     *
+     * @access private
+     * @memberOf BluetoothContext
+     * @param {String} address
+
+     * @param {Object} options
+     * @param {String} options.paired - is paired or not
+     * @param {String} options.address - the address of the device
+     * @return {Object} device
+     */
+    _findDeviceByAddress:
```

Comment:
I was wondering that the found out device in list is not our expected one. Such as: We want to find a device in paired devices list. But it's possible to find out it in remote devices list. If `ondevicefound`, `ondeviceunpaired`, `ondevicepaired` callback is coming (sequence/item)un-normally, I think we might update the device info in wrong list. That is why I have to design the function given remote/paired devices parameter. Then, try to find the device by address.